frontend/public/components/alert-manager.tsx

```diff
@@ -83,37 +84,72 @@ export const AlertManagersListContainer = props => <Firehose resources={[{
   <AlertManagersNameList {...props} />
 </Firehose>;
 
-const AlertManagerRow = ({obj: alertManager}) => {
+const tableColumnClasses = [
+  classNames('pf-m-2-col-on-lg', 'pf-m-3-col-on-md', 'pf-m-6-col-on-sm'),
+  classNames('pf-m-2-col-on-lg', 'pf-m-3-col-on-md', 'pf-m-6-col-on-sm'),
+  classNames('pf-m-3-col-on-lg', 'pf-m-3-col-on-md', 'pf-m-hidden', 'pf-m-visible-on-md'),
+  classNames('pf-m-2-col-on-lg', 'pf-m-hidden', 'pf-m-visible-on-lg'),
+  classNames('pf-m-3-col-on-lg', 'pf-m-3-col-on-md', 'pf-m-hidden', 'pf-m-visible-on-md'),
+];
+
+export const AlertManagerTableRow: React.FC<AlertManagerTableRowProps> = ({obj: alertManager, index, key, style}) => {
   const {metadata, spec} = alertManager;
+  return (
+    <Vr id={alertManager.metadata.uid} index={index} trKey={key} style={style}>
+      <Vd className={tableColumnClasses[0]}>
+        <ResourceLink kind={referenceForModel(AlertmanagerModel)} name={metadata.name} namespace={metadata.namespace} title={metadata.uid} />
+      </Vd>
+      <Vd className={tableColumnClasses[1]}>
+        <ResourceLink kind="Namespace" name={metadata.namespace} title={metadata.namespace} />
+      </Vd>
+      <Vd className={tableColumnClasses[2]}>
+        <LabelList kind={AlertmanagerModel.kind} labels={metadata.labels} />
+      </Vd>
+      <Vd className={tableColumnClasses[3]}>
+        {spec.version}
+      </Vd>
+      <Vd className={tableColumnClasses[4]}>
+        <Selector selector={spec.nodeSelector} kind="Node" />
+      </Vd>
+    </Vr>
+  );
+};
+AlertManagerTableRow.displayName = 'AlertManagerTableRow';
+export type AlertManagerTableRowProps = {
```

Comment:
This does not need to be exported (here and in the other `type` declarations in this PR).

Reply:
Hmm I did this so they could be used in tests if desired (and I believe some are currently). I can remove them though.

Reply:
I would add the `export` when it's used. If there are already tests that use it, feel free to leave it in.

Reply:
fixed in https://github.com/openshift/console/pull/1465/commits/6e61b79af85a9fe944a93665a59d3418e9b958d5